setup.py

```diff
@@ -41,11 +41,11 @@
     long_description_content_type="text/x-rst",
     include_package_data=True,
     keywords="dp_mobility_report",
-    name="dp-mobility-report",
+    name="dp_mobility_report",
```

Comment:
why change the name?

Reply:
To be consistent with the source package name

Reply:
This was the name that was automatically created from cookie-cutter based on our package name. I think it is also more common to use "-" for the name (e.g., scikit-learn https://pypi.org/project/scikit-learn/)
Might be better though to change the name when installing the package to have no "_" (e.g, as `import sklearn`)
maybe:
- dpmobility
- dpmobrep
- dpmreport
-
... ?